domains/smartmetering/smartmeteringws/configuration/setpushsetupalarm.md

```diff
@@ -2,7 +2,11 @@
 
 ## Description
 
-SetPushSetupAlarm is a request to define the destination of the TCP message that is optionally sent by the device. The request needs the DeviceIdentification, Host URL and port.
+SetPushSetupAlarm is a request to define the TCP message that is optionally sent by the device. The request consists of:
+- The destination: the DeviceIdentification, Host URL and port.
```

Comment:
Is `DeviceIdentification` part of the destination?

Reply:
No, I've changed the description so DeviceIdentification is not part of the destination anymore.